rootfs/opt/qserv/gcp/README.md

```diff
@@ -0,0 +1,15 @@
+# Create a GKE cluster for Qserv
+
+```shell
+# Create env.sh symlink to desired configuration
+ln -s env.qserv-cluster.sh env.sh
```

Comment:
Why would we need a symlink?

Reply:
It's a way to easily know the current configuration and to switch between configuration. env.sh can poins to env.qserv-cluster.sh or env.qserv-fjammes.sh or something else...